sesansdemo.py

```diff
@@ -5,81 +5,82 @@
 
 from __future__ import division
 
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
For readability I would prefer that basic math operators be imported from numpy: 
```
from numpy import pi, sqrt, sin, cos, exp, log
```
I care less about vector symbols such as linspace, zeros, zeros_like and meshgrid.